src/Symfony/Component/Lock/Store/ZookeeperStore.php

```diff
@@ -45,11 +45,15 @@ public static function createConnection(string $dsn): \Zookeeper
         }
 
         $host = $params['host'] ?? '';
-        if (isset($params['port'])) {
-            $host .= ':'.$params['port'];
+        $hosts = explode(',', $host);
+
+        foreach ($hosts as $index => $host) {
+            if (isset($params['port'])) {
+                $hosts[$index] = $host.':'.$params['port'];
+            }
         }
 
-        return new \Zookeeper($host);
+        return new \Zookeeper(implode($hosts, ','));
```

Comment:
please use the right signature of `implode` which has the separator first, instead of the legacy signature that is deprecated in 7.4 and removed in 8.0